packages/web-components/src/components/tabs-extended/tab.ts

```diff
@@ -72,21 +90,56 @@ class DDSTab extends StableSelectorMixin(LitElement) {
     super.updated(_changedProperties);
   }
 
-  render() {
+  protected _renderAccordionItem() {
+    const { label, selected, disabled, _index: index } = this;
+    const classes = classMap({
+      'bx--accordion__item': true,
+      'bx--accordion__item--active': selected,
+      'bx--accordion__item--disabled': disabled,
+    });
+    return html`
+      <li class="${classes}">
+        <button
+          class="${prefix}--accordion__heading"
+          aria-expanded="${selected}"
+          aria-controls="pane-${index}"
+          tabindex="${index + 1}"
+          ?disabled="${disabled}">
+          ${ChevronRight20({
+            part: 'expando-icon',
+            class: `${prefix}--accordion__arrow`,
+          })}
+          <div class="${prefix}--accordion__title">${label}</div>
+        </button>
+        <div id="pane-${index}" class="${prefix}--accordion__content">
+          <slot></slot>
+        </div>
+      </li>
+    `;
+  }
```

Comment:
This is a great refactor - I hadn't thought about making the tab responsible for the accordion functionality. 👏